compiler/fir/analysis-tests/testData/resolve/extendedCheckers/SafeInitializationChecker/delegation.kt

```diff
@@ -0,0 +1,5 @@
+class Hello {
+    val message: String by lazy { name }
+    <!UNREACHABLE_CODE!>val len = message.length<!>
```

Comment:
Why this code is unreachable?

Reply:
because calling `message` will `throw` here as `name` is not yet initialized :)

Reply:
This is my mistake again, since the prototype doesn't support delegates, I didn't pay attention to it.
This is due to the missing `// WITH_STDLIB`.

Reply:
@whyoleg, In fact, kotlin won't be able to detect that the "name" hasn't been initialized yet. To do this, we are trying to create an initialization analysis.

Moreover, when initializing the message property via name, you won't get an exception. The message will just be null